src/features/bookings/components/TicketBody/Ean/Ean.tsx

```diff
@@ -0,0 +1,27 @@
+import { t } from '@lingui/macro'
+import React, { FunctionComponent } from 'react'
+import styled from 'styled-components/native'
+
+import { Typo, getSpacing } from 'ui/theme'
+
+type Props = {
+  isbn?: string
```

Comment:
si on a pas d'ISBN, à quoi ça sert d'afficher l'EAN ?

```suggestion
  isbn: string
```

Reply:
Je suis d'accord. En théorie, le composant ne s'affiche pas car dans `BookingDetailsTicketContent` il y a cette condition d'affichage 
`  const shouldDisplayEAN = extraData?.isbn && categoryId === CategoryIdEnum.LIVRE`

Mais je typing `extraData?.isbn` ne me laisse pas tranquille dans `BookingDetailsTicketContent`